policies/yaml_schema.rego

```diff
@@ -48,6 +60,18 @@ deny[yaml_schema_violation(description, group.id, name)] {
     description := sprintf("Event name '%s' is invalid. Event name %s'", [name, invalid_name_helper])
 }
 
```

Comment:
Should (or do we?) have a test that ensures now that resource `id` is in the form of `resource.{name}`?

Reply:
great catch! I had tests for spans, metrics and events, but not for resources